src/dataDisplay/personCoreMilestones/gridColumnAcceptedChrist.jsx

```diff
@@ -0,0 +1,157 @@
+    REMOVE_ACCEPTED_CHRIST_COLUMN_DEFAULT_PROP,
+    REMOVE_ACCEPTED_CHRIST_COLUMN_PROP_TYPE,
+} from './constants';
+import Grid from '../../layout/grid';
+import Icon from '../icon';
+import makeStyles from '../../styles/makeStyles';
+import MilestonePopoverContent from './milestonePopoverContent';
+import Popover from '../popover';
+
+const propTypes = {
+    acceptedChristDate: ACCEPTED_CHRIST_DATE_PROP_TYPE,
+    classes: PropTypes.shape({
+        icon: PropTypes.string,
+        root: PropTypes.string,
+    }).isRequired,
+    hasAcceptedChrist: HAS_ACCEPTED_CHRIST_PROP_TYPE,
+    iconSize: ICON_SIZE_PROP_TYPE,
+    inverse: INVERSE_PROP_TYPE,
+    isFemale: PropTypes.bool,
+    isMale: PropTypes.bool,
+    recordType: RECORD_TYPE_PROP_TYPE,
+    removeAcceptedChristColumn: REMOVE_ACCEPTED_CHRIST_COLUMN_PROP_TYPE,
+};
+
+const defaultProps = {
+    acceptedChristDate: null,
+    hasAcceptedChrist: null,
+    iconSize: ICON_SIZE_DEFAULT_PROP,
+    inverse: INVERSE_DEFAULT_PROP,
+    isFemale: PropTypes.bool,
+    isMale: PropTypes.bool,
+    recordType: RECORD_TYPE_PROP_TYPE,
+    removeAcceptedChristColumn: REMOVE_ACCEPTED_CHRIST_COLUMN_DEFAULT_PROP,
+};
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        '& $hasAcceptedChrist': {
+            '&$isAdult': {
+                '&$genderFemale .icon-use-path': {
+                    fill: `${RECORD_TYPE_COLOR({ gender: 'f', recordType: 'adult', theme })}`,
+                },
+                '&$genderMale .icon-use-path': {
+                    fill: `${RECORD_TYPE_COLOR({ gender: 'm', recordType: 'adult', theme })}`,
+                },
+                '&$genderUndefined .icon-use-path': {
+                    fill: `${RECORD_TYPE_COLOR({ recordType: 'adult', theme })}`,
+                },
+            },
+            '&$isChild .icon-use-path': {
+                fill: `${RECORD_TYPE_COLOR({ recordType: 'child', theme })} !important`,
+            },
+            '&$isStudent .icon-use-path': {
+                fill: `${RECORD_TYPE_COLOR({ recordType: 'student', theme })} !important`,
+            },
+        },
+    },
+    genderFemale: {},
+    genderMale: {},
+    genderUndefined: {},
+    hasAcceptedChrist: {},
+    icon: {},
+    isAdult: {},
+    isChild: {},
+    isStudent: {},
+}));
+
+function GridColumnAcceptedChrist(props) {
+    const {
+        acceptedChristDate,
+        hasAcceptedChrist,
+        iconSize,
+        inverse,
+        isFemale,
+        isMale,
+        recordType,
+        removeAcceptedChristColumn,
+    } = props;
+
+    const classes = useStyles(props);
+
+    if (removeAcceptedChristColumn) {
+        return null;
+    }
+
+    const popoverContentAcceptedChrist = (hasAcceptedChrist && acceptedChristDate) ? (
+        <MilestonePopoverContent
+            title="Accepted Christ"
+            milestonesDates={[
+                { label: 'On', date: moment(acceptedChristDate).format() },
+            ]}
+        />
+    ) : '';
```

Comment:
Why empty string rather than `null`?  Not Blocking; just a question.
Applies to all similar placements in various other `gridColumnXyz.jsx` files here.

Reply:
Looks like we updated it to `null`.  Resolving.